setup/releaseVersionUpdate.py

```diff
@@ -0,0 +1,48 @@
+import json
+import requests
+import sys
+import os
+from xml.dom import minidom
+from azure.storage.blob import BlobServiceClient, BlobClient, ContainerClient
+
+
+sasUrl = sys.argv[1]
+releaseVersion = sys.argv[2].split(' ')[2]
+print('Release Version: ' + releaseVersion)
+if(len(releaseVersion)==0):
+    print('Incorrect Release Version')
+    sys.exit(1)
+
+containerUrl = sasUrl.split('?')[0]
+sasToken = sasUrl.split('?')[1]
+
+# delete latest version file in the container
+deleteUrl = containerUrl + '/' + 'releasemetadata/latest_version.txt' + '?' + sasToken
```

Comment:
I believe this should be containerURL + '/latest_version.txt'

The current URL results in the wrong file path.

Reply:
sure will correct it.

Reply:
corrected